ui/src/testing/stubs/workspaces.tsx

```diff
@@ -61,6 +61,11 @@ export function buildWorkspaceStub(suffix = ''): Workspace {
     billingAccountName: 'billing-account',
     billingStatus: BillingStatus.ACTIVE,
     adminLocked: false,
+    initialCredits: {
```

Comment:
Workspace-about tests did not have initial credits by default, so I added defaults to the stub.